odoo/addons/base/models/res_partner.py

```diff
@@ -204,6 +204,7 @@ def default_get(self, default_fields):
     email_formatted = fields.Char(
         'Formatted Email', compute='_compute_email_formatted',
         help='Format email address "Name <email@domain>"')
+    phone_mobile_search = fields.Char("Phone/Mobile", store=False, search='search_phone_mobile_search')
```

Comment:
I guess you will run into troubles. This field is not available in base, only when having phone_validation installed.

Reply:
So what do you propose as a solution? I am thinking about creating a mixin in base and make crm_lead and res_partner inherit from it. This way it will be accessible even if phone_validation is not installed.

Reply:
That's not how it works. Base give access to some base behavior. Phone advanced management comes with phone_validation module. It would not make sense to have access to an advance phone search while you don't have any tool available for formatting or sanitizing phone numbers. If you look a bit at module dependencies it is quite straightforward to find out where to put that code and partner view update :) . Don't you have any idea ?

Reply:
Yeah after taking a deeper look, I think that putting the field in the res_partner model located in the phone_validation module would make much more sense. Is that right?

Reply:
Indeed :) . CRM already depends on phone_validation so nothing to change there (except moving code).